src/components/ClickableMap/ClickableMap.js

```diff
@@ -198,50 +205,106 @@ function ClickableMap({ onLocationChange, locationInfo }) {
     }
   }
 
+  function handlePlaceChange(address) {
+    setCurrentPlaceLabel(address);
+  }
+
+  function handlePlaceSelect(_event, selection) {
+    if (!selection) return;
+
+    geocodeByAddress(selection.description)
```

Comment:
Did you do promise chaining over async/await here for a particular reason? i.e.

```js
async function handlePlaceSelect(_event, selection) {
  if (!selection) return;
  try {
    const [address] = await geocodeByAddress(selection.description)
    const { lat: latitude, lng: longitude } = await getLatLng(address)
    map.setZoom(15);
    setLocation({ latitude, longitude });
  } catch(err) {
    showError('Failed to get the location from address.');
    // eslint-disable-next-line no-console
    console.error('Failed to get the location from address:', error.message);
  }
}
```

Reply:
Ahh - just noticed the docs of react-places-autocomplete uses the chaining in there example

Reply:
Updated